log.go

```diff
@@ -251,6 +255,12 @@ func (l Logger) Level(lvl Level) Logger {
 	return l
 }
 
+// Section creates a child logger filtering messages based on the existence of the provided section.
+func (l Logger) Section(section string) Logger {
```

Comment:
Instead of storing the section, you could test the section right away, and either return the same logger or a disabled logger depending on the result.

Reply:
Sure. 
But doing so prevents to dynamically change the configuration of sections at running time. I like this feature but I presume the issue of performance is more important ?

Reply:
Ah yes, but implementation has the following issue: you can't call `Section()` inline as it returns a `logger` and not a `*logger` as expected by the sub-functions of the call chain (like `Debug()`). This is the same function  signature than `Level()` (I've worked by imitation :smile: ).

Reply:
That's easy to fix :)